lib_test/dune

```diff
@@ -1,6 +1,10 @@
 (executables
  (names test_runner)
- (libraries uri uri.sexp uri_services oUnit sexplib0))
+ (modules test_runner uri_sexp)
+ (preprocess (pps ppx_sexp_conv))
+ (libraries uri uri_services oUnit sexplib0))
+
+(rule (copy# ../lib/uri_sexp.ml uri_sexp.ml))
```

Comment:
uhm, can we find a better solution than `copying` modules around for testing? depending on the test case shape, maybe they should only be run for uri-sexp package?

Reply:
This is, indeed, the hard part of this PR and I did not find an other solution that this ... I will retry something this week.